md.py

```diff
@@ -154,10 +159,11 @@ def setup_from_args(self, args=None):
                             help="md chunk size")
 
         group = parser.add_mutually_exclusive_group()
-        group.add_argument("-d", "--disks", "--ram-disks", type=int,
+        group.add_argument("-d", "--disks", type=int, default=3,
                            help="number of disks to create")
-        group.add_argument("--loop-disks", type=int,
-                           help="use loop devices instead of ram disks")
+        group.add_argument("--disk-type", default="default",
+                           choices=["default", "ram", "dev", "loopback"],
```

Comment:
Do we really the "default" option? We can just check if disk_type is unset and then decide if we need to use ram or dev